### ISSUE TITLE
Angular 2

### ISSUE THREAD

**Issue by arcanewater** [ibbatta/grunt-frontend-boilerplate#21]
Hey thanks for this boilerplate, any plans to upgrade angular to angular 2?

**Comment by ibbatta** [comment]
Hi @arcanewater , thanks for your request.
Actually I am studying some best practice to evolve this project in Angular 2 (but in my spare time, so it will take long time)

**Comment by bampakoa** [comment]
@ibbatta Do you have any thoughts on that? Do you want to maintain a different branch or replace the master one?

**Comment by ibbatta** [comment]
@bampakoa I am thinking on (actually I am more focussed on React and OOP in javascript, but I am still keeping an eye on this issue... I really want learn Angular 4 for the purpose)

**Comment by ibbatta** [comment]
@bampakoa thanks I really appreciate your help.
How are you willing to go on it?

P.s. is there any yeoman generator for Angular 4?

**Comment by bampakoa** [comment]
If you do not have any objection (because of the **grunt** word in the name of the repository :smile: ) I will probably go with the Angular CLI which has really good automations. I do not have a clue about yeoman generator for Angular 4 :thinking:

**Comment by ibbatta** [comment]
@bampakoa yeah 😄 it was the main reason of this repo! I know that it's hard for Angular 2+ so it's ok for me to use the Angular CLI... but what will the main purpose of the repo then? (maybe a tutorial to learn Angular 4? 🤔 )

**Comment by bampakoa** [comment]
Let's keep it the **grunt** way. It will be more fun and challenging for me :smiley:

**Comment by ibbatta** [comment]
@bampakoa we could have a try before pass to Angular CLI, why not :)
